tests/trace/test_client_trace.py

```diff
@@ -3236,3 +3236,87 @@ def test():
 
     assert len(test.calls()) == 2
     assert len(client.get_calls()) == 2
+
+
+@pytest.mark.parametrize(
+    "descendents_code",
+    [
+        "list(call.get_descendents())",
```

Comment:
this is pretty nasty. If you really want to parameterize this, pass in functions, not a raw eval